pandasai/__init__.py

```diff
@@ -323,7 +323,10 @@ def _is_unsafe_import(self, node: ast.stmt) -> bool:
                 if alias.name in WHITELISTED_OPTIONAL_LIBRARIES:
                     import_optional_dependency(alias.name)
                     continue
-                if alias.name not in WHITELISTED_LIBRARIES:
+                if not any(
+                    alias.name.startswith(whitelisted_lib)
```

Comment:
This is scary, cause if a library whose name starts with another whitelisted library, we would allow it to be imported. This could cause some injections which we ideally want to avoid

Reply:
yes, i though the same therefore explicitly defined the module in whitelist. This is extended based on the suggestion here: https://github.com/gventuri/pandas-ai/pull/211#discussion_r1214895423

Reply:
What if instead of `alias.name.startswith()` we look for `alias.name.split('.')[0]`?

Reply:
@sandiemann @jonbiemond I'd definitely go for it! Can anyone see any cons?

Reply:
@gventuri I've added the changes as suggested [here](https://github.com/gventuri/pandas-ai/pull/211/commits/95f2820afdfbfa4f4ef92205891a5e969927569b) Please review